pkg/cmd/admin/groups/new.go

```diff
@@ -49,7 +49,17 @@ func NewCmdNewGroup(name, fullName string, f *clientcmd.Factory, out io.Writer)
 				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
-			kcmdutil.CheckErr(options.AddGroup())
+			if users, err := options.AddGroup(); err != nil {
+				kcmdutil.CheckErr(err)
+			} else {
+				var msg string
+				if len(users) == 0 {
+					msg = fmt.Sprintf("Group %q created.", options.Group)
+				} else {
+					msg = fmt.Sprintf("Group %q created with users %q.", options.Group, users)
```

Comment:
@deads2k the original intent of my issue was that your command didn't make use of the success printer that we were moving to standardize around at the time